### PR TITLE
Added support for Xcode 8 and iOS 10

### PR DESCRIPTION
**Pull request by tzraikov**

### CODE REVIEW

**Comment by dtopuzov** [comment]
:100:

**Comment by enchev** [comment]
:+1:

File: lib/services/ios-project-service.ts [hunk 1]
```diff
@@ -293,8 +293,8 @@ export class IOSProjectService extends projectServiceBaseLib.PlatformProjectServ
 			} else {
 				args = basicArgs.concat([
 					"-sdk", "iphonesimulator",
-					"-arch", "i386",
-					'VALID_ARCHS="i386"',
+					"-arch", "x86_64",
```
Comment (rosen-vladimirov) [inline lib/services/ios-project-service.ts]: will this work with XCode 7 ?

Comment (tzraikov) [inline lib/services/ios-project-service.ts]: Tested on Xcode 7.3.1

**Pull request merged**